README.md

```diff
@@ -1,4 +1,21 @@
 # sidecore
 
 ![Sidecore](art/sidecorelogo.png)
-sidecore lets you run IoT systems, using flattened docker images from githu.bcom/u-root/sidecore-images, as easily as you run a shell script
+
+Using flattened Docker images from <https://github.com/u-root/sidecore-images>,
+**sidecore** lets you run IoT systems as easily as you run a shell script.
+
+## How it works
+
+Building on top of [`cpu`](https://github.com/u-root/cpu), **sidecore** merely
+requires your IoT system to run a small daemon in order to use it as a resource.
+
+## Building
+
+**NOTE**: Go version 1.20 is required as a minimum.
+
+Run `go build ./cmds/sidecore` to build the command.
```

Comment:
might want to add:

go install github.com/u-root/sidecore/cmds/sidecore@latest

I just tried this and it works fine.

Reply:
I will add that in a subsequent PR.